app/models/automated_checks/claim_verifiers/identity.rb

```diff
@@ -18,16 +18,18 @@ def perform
         return unless awaiting_task?(TASK_NAME)
 
         # Order of matching matters so that subsequent conditions in methods fall through to execute the right thing
-        no_match ||
-          partial_match ||
-          complete_match
+        auto_pass || no_match || partial_match || complete_match
       end
 
       private
 
       attr_accessor :admin_user, :claim
       attr_reader :dqt_teacher_status
 
+      def auto_pass
+        create_task(match: nil, passed: true) if claim.policy.further_education_payments?
```

Comment:
I think we should iterate the `Policies::POLICIES` and let them determine this. We have a few instances elsewhere were policy specific logic is hardcoded and has been missed.